tests/samplers_tests/test_samplers.py

```diff
@@ -58,7 +58,12 @@
     + (
         []
         if sys.version_info < (3, 7, 0)
-        else [lambda: optuna.integration.BoTorchSampler(n_startup_trials=0)]
+        else [
+            pytest.param(
+                lambda: optuna.integration.BoTorchSampler(n_startup_trials=0),
+                marks=pytest.mark.slow,
+            )
+        ]
```

Comment:
Except for the reproducibility test, please keep testing the `BoTorchSampler`.

Reply:
I updated the codes to skip only reproducible tests and `reseed_eng` tests. PTAL.